tests/test_transpile.py

```diff
@@ -142,6 +142,7 @@ def test_time(self):
             "FROM_UNIXTIME(123)",
             write='spark',
         )
+        self.validate("CREATE TEMPORARY TABLE test AS SELECT 1", "CREATE TEMPORARY VIEW test AS SELECT 1", write='spark')
```

Comment:
does this work for the identity case? can you put an identity test

create temporary table test in identity sql? 

that seems like it would fail since you haven't added it

Reply:
Fixed and added test cases.